oat-sa/extension-tao-itemqti
# Feature/tao 3273 eliminate choice

TAO-3273

Test taker can now eliminate choices, made available for the whole platform

Test Instructions
- create a choice interaction
- select 'allow elimination'
- try this also in connection with list styles

Acceptance criteria
- I can press a button that causes answers to be eliminated.
- eliminated answers can be 'un eliminated' by a second button press
- eliminated answers cannot be selected as correct

Note that in Edge the background of the checkboxes is not in the right place. This is a known issue and dealt with in another PR. You can ignore this for now.
